include/crpropa/module/PhotoPionProduction.h

```diff
@@ -12,13 +12,20 @@ namespace crpropa {
  * @{
  */
 
+struct SophiaEventOutput {
```

Comment:
OK, I meant a vector of structs. But for a temporary solution this will be ok.

Reply:
I could implement such a structure and it would compile properly and could be used in a terminal window as intended. However, I could not access the output data due to SWIG deeming the output data non-iterable.